src/mlpack/tests/main.cpp

```diff
@@ -22,8 +22,8 @@ int main(int argc, char** argv)
    * each run.  This is good for ensuring that a test's tolerance is sufficient
    * across many different runs.
    */
-  // size_t seed = std::time(NULL);
-  // mlpack::math::RandomSeed(seed);
+  size_t seed = std::time(NULL);
+  mlpack::math::RandomSeed(seed);
```

Comment:
```suggestion
  // size_t seed = std::time(NULL);
  // mlpack::math::RandomSeed(seed);
```

Reply:
Thanks!  Forgot to recomment that after testing.